src/components/shared/chip/chip.styles.ts

```diff
@@ -27,14 +27,14 @@ export const ChipButton = styled.button<IChipButtonProps>`
 	${(props) => {
 		if (props.isActive) {
 			return css`
-				background-color: ${V2_Color.Neutral[4](props)};
+				background-color: ${Colour["bg-inverse-subtlest"](props)};
 
 				${ChipText} {
```

Comment:
actually, this hasn't been working as the component has been using the typography component directly all this while. update the component to use `ChipText` instead